Requirements/ViewModels/RequirementBrowser/RequirementsBrowserViewModel.cs

```diff
@@ -747,6 +754,13 @@ public async void ExecuteVerifyRequirements(Thing option)
 
                     var targetOption = option as Option;
 
+                    ElementDefinition topElement = ((Iteration)option.Container).TopElement;
+                    if (topElement == null)
+                    {
+                        this.messageBoxService.Show("Requirements can't be verified.\nTop Element is not set.", "Verification failed", MessageBoxButton.OK, MessageBoxImage.Error);
```

Comment:
i propose we change 
  - the message to: Verification not performed.
  - the icon from error to something more friendly like warning or information